src/test/java/core/basesyntax/service/impl/WriterImplTest.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.Writer;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Collections;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class WriterImplTest {
+    private static Writer writer;
+
+    @BeforeClass
+    public static void setUp() {
+        writer = new WriterImpl();
+    }
+
+    @Test
+    public void write_validData_ok() {
+        // expected
+        String content = "testContent123  .. to add" + System.lineSeparator()
+                + "second line of test writer" + System.lineSeparator()
+                + "the last one is here";
+        File expectedFile = new File("src/test/resources/writer_validDataExpected.csv");
+        try {
+            Files.write(expectedFile.toPath(), Collections.singleton(content));
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t write data to file " + expectedFile.getName(), e);
+        }
+        List<String> expected;
+        try {
+            expected = Files.readAllLines(expectedFile.toPath());
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t read data from file " + expectedFile.getName(), e);
+        }
+
+        // actual
+        File actualFile = writer.write(content, "src/test/resources/writer_validDataActual.csv");
```

Comment:
let's just write once with our writerService
and then read with Files.readAllLines()